### DISCUSSION TITLE
Entering Viewer Mode through Scripts

### DISCUSSION THREAD

**Discussion by ninarular** [cwapi3d/cwapi3dpython#189] [General]
Similar to what the title asks. is there a way to enter the Viewer mode model of a selected container element through code? I would like to export .sat Files however the elements need to display the connector axis holes that are shown in viewer mode

**Comment by Brunner246** [comment]
Hi,
Entering the viewer mode via API will not be implemented, but we will create a function to export `.sat` files with "real" geometry.
